themes/default/content/registry/packages/aws/how-to-guides/v6-migration.md

```diff
@@ -0,0 +1,587 @@
+- aws:cloudtrail/getFunction:getFunction -> [aws:cloudfront/getFunction:getFunction](https://www.pulumi.com/registry/packages/aws/api-docs/cloudfront/getfunction/)
+- aws:elasticloadbalancing/getHostedZoneId:getHostedZoneId -> [aws:elb/getHostedZoneId:getHostedZoneId](https://www.pulumi.com/registry/packages/aws/api-docs/elb/gethostedzoneid)
+- aws:elasticloadbalancing/getLoadBalancer:getLoadBalancer -> [- aws:elb/getLoadBalancer:getLoadBalancer](https://www.pulumi.com/registry/packages/aws/api-docs/elb/getloadbalancer)
+- aws:elasticloadbalancing/getServiceAccount:getServiceAccount -> [- aws:elb/getServiceAccount:getServiceAccount](https://www.pulumi.com/registry/packages/aws/api-docs/elb/getserviceaccount)
+- aws:elasticloadbalancingv2/getListener:getListener -> [aws:lb/getListener:getListener](https://www.pulumi.com/registry/packages/aws/api-docs/lb/getlistener)
+- aws:elasticloadbalancingv2/getLoadBalancer:getLoadBalancer -> [aws:lb/getLoadBalancer:getLoadBalancer](https://www.pulumi.com/registry/packages/aws/api-docs/lb/getloadbalancer)
+- aws:elasticloadbalancingv2/getTargetGroup:getTargetGroup -> [aws:lb/getTargetGroup:getTargetGroup](https://www.pulumi.com/registry/packages/aws/api-docs/lb/gettargetgroup)
+- aws:iam/getOpenidConnectProvider:getOpenidConnectProvider -> [aws:iam/getOpenIdConnectProvider:getOpenIdConnectProvider](https://www.pulumi.com/registry/packages/aws/api-docs/iam/getopenidconnectprovider/)
+- aws:index/getAmi:getAmi -> [aws:ec2/getAmi:getAmi](https://www.pulumi.com/registry/packages/aws/api-docs/ec2/getami/)
+- aws:index/getAmiIds:getAmiIds -> [aws:ec2/getAmiIds:getAmiIds](https://www.pulumi.com/registry/packages/aws/api-docs/ec2/getamiids/)
+- aws:index/getAutoscalingGroups:getAutoscalingGroups -> [aws:autoscaling/getAmiIds:getAmiIds](https://www.pulumi.com/registry/packages/aws/api-docs/autoscaling/getamiids/)
+- aws:index/getCanonicalUserId:getCanonicalUserId -> [aws:s3/getCanonicalUserId:getCanonicalUserId](https://www.pulumi.com/registry/packages/aws/api-docs/s3/getcanonicaluserid/)
+- aws:index/getElasticIp:getElasticIp -> [aws:ec2/getElasticIp:getElasticIp](https://www.pulumi.com/registry/packages/aws/api-docs/ec2/getelasticip/)
+- aws:index/getPrefixList:getPrefixList -> [aws:ec2/getPrefixList:getPrefixList](https://www.pulumi.com/registry/packages/aws/api-docs/ec2/getprefixlist/)
+
+## [Typescript] `aws.sdk` Property removed
+
+The [aws.sdk](https://github.com/pulumi/pulumi-aws/pull/2584)
+property has been removed as it has already been deprecated upstream. The underlying functionality is still accessible by directly importing the AWS Typescript sdk. See https://github.com/pulumi/pulumi-aws/pull/2584 for more details.
+
+```patch
+- import * as aws from "@pulumi/aws";
++ import { Bucket } from "@pulumi/aws/s3";
++ import * as s3 from "@aws-sdk/client-s3";
+
+-const bucket = new aws.s3.Bucket("my-bucket");
++const bucket = new aws.s3.Bucket("my-bucket");
```

Comment:
Are these supposed to be different?

Reply:
Yes. Fixed in https://github.com/pulumi/registry/pull/2941/commits/bdf97cd05ad61fdc4e4cef3da9ce8678fc278536.